utils/fetchers/fetcher.rb

```diff
@@ -0,0 +1,38 @@
+module Util
```

Comment:
module name is `Util` but directory name is `utils`. I'd like to see the module name be `Utils`, unelss you had a particular reason to do otherwise.